### ISSUE TITLE
Test "testStreamersSuccessful" fails sometimes

### ISSUE THREAD

**Issue by Vaalyn** [AzuraCast/php-api-client#13]
Sometimes the `testStreamersSuccessful` test fails due to invalid response from AzuraCast.

Needs to be investigated to determine the root cause.

```
1) AzuraCast\Api\AzuraCastApiClientTest::testStreamersSuccessful
TypeError: Argument 1 passed to AzuraCast\Api\Dto\StreamerDto::fromArray() must be of the type array, null given, called in /app/src/Station/StreamersClient.php on line 88
/app/src/Dto/StreamerDto.php:233
/app/src/Station/StreamersClient.php:88
/app/tests/AzuraCastApiClient/AzuraCastApiClientTest.php:488
```

**Comment by Vaalyn** [comment]
The error stems from the `POST` call to setup a streamer account. The password generated by faker contains an invalid character for that entity.

**Comment by Vaalyn** [comment]
Closed by PR #12
